query-node/mappings/src/common.ts

```diff
@@ -0,0 +1,82 @@
+import BN from 'bn.js'
+import { SubstrateEvent } from '@dzlzv/hydra-common'
+import { DatabaseManager } from '@dzlzv/hydra-db-utils'
+import { u64 } from '@polkadot/types/primitive';
+
+import { Network } from 'query-node/src/modules/enums/enums'
+
+// Asset
+import {
+  DataObjectOwner,
+} from 'query-node/src/modules/variants/variants.model'
+import {
+  DataObject,
+  LiaisonJudgement,
+} from 'query-node/src/modules/data-object/data-object.model'
+import {
+  ContentParameters
+} from '@joystream/types/augment'
```

Comment:
Type `ContentParameters` from `@joystream/types/augment` corresponds to the actual object received from processor. It needs to be cast into custom Joystream object like this:
```
import {
  ContentParameters,
} from '@joystream/types/augment'

import { ContentParameters as Joystream_ContentParameters } from '@joystream/types/storage'
import { registry } from '@joystream/types'

...
function doIt(contentParameters: ContentParameters) {
    const joystreamContentParameters = new Joystream_ContentParameters(registry, contentParameters.toJSON() as any)
    ...
}
```
Otherwise, getters like `size_in_bytes` and similar will not be available.